### ISSUE TITLE
this plugin doesn't seem to work along with "trilbymedia/grav-plugin-image-captions"

### ISSUE THREAD

**Issue by lustyffh** [sal0max/grav-plugin-shortcode-gallery-plusplus#8]
Hello, I have your plugin with default configuraiton.
When I have "trilbymedia/grav-plugin-image-captions" plugin enabled and reference an image with 'caption' class in my page, i.e.

`![](image1.jpg?classes=caption,center "My image caption")`

Then any [gallery] within the same page is not rendered anymore.

The source section becomes just empty:

```
<p id="6977931">
    </p>
....
    <script src="/assets/5246ceab977a56b766085feb05e6a042.js"></script>
<script>
$("#6977931").justifiedGallery({
    rowHeight: 400,
    margins: 8,
    lastRow: "nojustify",
    captions: false,
    border: 0,
});

GLightbox({
    selector: ".glightbox-6977931",
    openEffect: "zoom",
    closeEffect: "fade",
    slideEffect: "slide",
    closeButton: true,
    touchNavigation: true,
    touchFollowAxis: true,
    keyboardNavigation: true,
    closeOnOutsideClick: true,
    loop: true,
    draggable: true,
    descPosition: "bottom",
});
</script>
```

While it still works if the same image comes only with the 'center' class selector.
`![](image1.jpg?classes=center "My image caption")`

Just in case - I use "quark-open-publishing" theme

**Comment by sal0max** [comment]
Thanks for the info.
I took a look at this behavior and found the following:

Without the _image-captions_ plugin intervening, a Markdown image link like `![](image.jpg)` gets converted to e.g.
```html
<img alt="" src="/user/pages/demo/image.jpg" />
```

With the _image-captions_ plugin the Markdown `![](image.jpg?classes=caption "Caption")` gets converted to:
```html
<figure class="image-caption">
   <img title="Caption" alt="" class="caption" src="/user/pages/demo/image.jpg">
   <figcaption class="">My Image Caption</figcaption>
</figure>
```

Besides the `figure`-stuff, there's a major difference:
* The _image-captions_ plugin generates a valid HTML5 `img` element **without** closing tag.
* Whereas the default Grav implementation adds a closing `/>`.

The Regex my plugin includes, expected the closing Tag to be there.


**So, tldr;:**
That's a quick fix I can do. Thanks for this issue.
Update is on its way.

**Comment by sal0max** [comment]
Fixed in 6cfda86341ffd0da663168b854581cd9534f9416

**Comment by lustyffh** [comment]
wow, thanks a lot for prompt feedback! I confirm it works now.
